apps/compute-cf.yml.j2

```
@@ -51,6 +51,8 @@ Resources:
     Type: AWS::EC2::LaunchTemplate
     Properties:
       LaunchTemplateData:
+        MetadataOptions:
+            HttpTokens: 'required'
```

We're using two-space indentation for yaml files, although I think this is syntactically valid either way. And I lean towards no using quotes for strings in yaml unless specifically required.
```suggestion
          HttpTokens: required
```